Sources/GRPCProtobuf/Documentation.docc/Articles/Code-generation-with-protoc.md

```diff
@@ -0,0 +1,97 @@
+# Code generation with protoc
+
+This article describes how to use the Swift Package Manager build plugin to
+generate gRPC Swift and Swift Protobuf code from your Protocol Buffers `.proto` files.
+
+## Overview
+
+If you've used Protocol Buffers before then generating gRPC Swift stubs should be simple. If you're
+unfamiliar with Protocol Buffers then you should get comfortable with the concepts before
+continuing; the [Protocol Buffers website](https://protobuf.dev/) is a great place to start.
+
+If you haven't installed `protoc` yet refer to <doc:Installing-protoc> for
+instructions.
+
+## Using protoc
+
+The [`grpc-swift-protobuf`](https://github.com/grpc/grpc-swift-protobuf) package provides
+`protoc-gen-grpc-swift-2`, a program which is a plugin for the Protocol Buffers compiler, `protoc`.
+To generate gRPC stubs for your `.proto` files directly you must run the `protoc` command with
+the `--grpc-swift-2_out=<DIRECTORY>` option:
+
+```console
+protoc --grpc-swift-2_out=. my-service.proto
+```
+
+> `protoc-gen-grpc-swift-2` only generates gRPC stubs, it doesn't generate messages. You must use
+> `protoc-gen-swift` to generate messages in addition to gRPC Stubs.
```

Comment:
I'm unsure about this, but should we maybe put this at the top of this section and slightly rephrase it to something like "you should first make sure you've generated your proto messages by using `protoc-gen-swift` before generating gRPC Stubs? 

I just feel like if you're completely new to all of this, this paragraph may be confusing?

Reply:
It doesn't matter which order they're generated in, it can be before or after generating gRPC stubs. I think this is in the right place because the paragraph above sets the scene by explaining that we provide a plugin for protoc which generates the gRPC bits. This is just making it more explicit that it only generates the gRPC bits.